eval-config.nix

```diff
@@ -24,6 +24,6 @@ in
 {
   inherit (eval._module.args) pkgs;
   inherit (eval) options config;
-
+  inherit _module;
```

Comment:
Did you mean?

```suggestion
  inherit (eval) _module;
```

Reply:
You're right. I could've sworn that it worked before, but since `nixos-unstable` got updated it blew up in my face.
Thanks!